chain/ethereum/src/network.rs

```diff
@@ -109,15 +117,14 @@ impl EthereumNetworkAdapters {
         let provider = ProviderManager::new(
             Logger::root(Discard, o!()),
             vec![(chain_id.clone(), adapters)].into_iter(),
-            Arc::new(NoopIdentValidator),
+            &[],
```

Comment:
from usability perspective I think I'd rather have a type you pass here like NoChecks or a AndChecks(Box<[]>) or OneOf(Box<[]>), in this case I think we'd only need the first 2 obviosuly. I think this probably more of a nitpick though